Mlem/Views/Tabs/Posts/Components/Post/Components/Community View/Post Item/Lazy Load Expanded Post.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  Lazy Load Expanded Post.swift
+//  Mlem
+//
+//  Created by Jake Shirey on 23.06.2023.
+//
+
+import SwiftUI
+
+struct LazyLoadExpandedPost: View
+{
+    @State var account: SavedAccount
+    @State var post: APIPost
+    
+    @State private var loadedPostView: APIPostView? = nil
+    
+    var body: some View {
+        ZStack {
+            if let loadedPost = loadedPostView {
+                ExpandedPost(account: account, post: loadedPost, feedType: .constant(.subscribed))
+            }
+            else {
+                progressView
+            }
+        }
+        .task(priority: .userInitiated) {
+            
```

Comment:
Hmmm looks like something went missing in here 🤔  (or maybe that's a leftover since there is a task later on)

Reply:
Whoops! Looks like leftover debug code.